lib/core/network/protocols/websocket.js

```diff
@@ -208,6 +208,12 @@ class WebSocketProtocol extends Protocol {
 
     try {
       parsed = JSON.parse(data);
+      if (parsed.p && parsed.p === 1) {
```

Comment:
You also need to verify that this is the only property of the received JSON payload.

For all we know, there might be a custom API action out there having a numeral `p` parameter...